### DISCUSSION TITLE
Youtube music no longer playing through bot

### DISCUSSION THREAD

**Discussion by Par4site** [codetheweb/muse#1041] [Q&A] [answered]
Greetings,
So I've noticed that at least the docker image for 2.8.1 is no longer playing youtube links. From what I can see, it's not even trying to cache anymore either, almost as if the youtube API changed or something to where it cannot reach youtube. I've even tried to make a new API key and use that no to avail.
I've also tried to make a new docker image with the same keys and stuff as well, seems to still not be working. There are no error messages or logs indicating that something is wrong, so I'm not sure where to go from here (bot happily says that music has been added to queue, and the docker logs just say that everything is kocher). It does, however, play locally added songs, so I know the discord keys are good, and the bot can at least play music. Any suggestions / guidance?

**Comment by PackLack197** [comment]
Same issue here too. Muse in my server isn't working with YouTube, /queue also says there's nothing in the queue.

**Comment by astrobound** [comment]
Im getting the same issue discribed to.
Consol spat this out
Error checking for updates: Status code: 403
You can disable this check by setting the `YTDL_NO_UPDATE` env variable.

**Reply by Par4site** [reply]
I'm not sure I see an environment variable for YTDL_NO_UPDATE. What values does it take, doing a quick search shows it can take 1 or true, but not sure which to use.

**Comment by Par4site** [comment, marked as answer]
Copied from issues post:
Maybe have a look at https://github.com/codetheweb/muse/issues/1037 that could help

TL:DR using latest has a broken dependency in it, pull docker image codetheweb/muse:pr-1042 instead of codetheweb/muse:latest and run that
